uelc/main/models.py

```diff
@@ -188,6 +174,22 @@ def __unicode__(self):
     def display_name(self):
         return '%s' % (self.name)
 
+    def _get_cohorts(self):
+        cohorts = Cohort.objects.filter(case_cohort=self.id)
+        if not cohorts:
+            return None
+        return cohorts
+
+    def cohortnames(self):
+        if self.cohort.all():
```

Comment:
I find this a little confusing .. where is `self.cohort` defined? Also, if it's a queryset I think it's clearer to do either `if self.cohort.exists()` or `if self.cohort.count() > 0`, if you're checking if any exist.